src/js/components/Calendar/Calendar.js

```diff
@@ -640,6 +640,8 @@ const Calendar = forwardRef(
     let day = new Date(displayBounds[0]);
     let days;
     let firstDayInMonth;
+    // if a week is filled with all blank days, we don't want to set role="row"
```

Comment:
Feel like this comment may be more helpful where the logic is implemented on L793 and this comment could either be eliminated or read
```suggestion
    // a week containing only blank days
```
if you feel like it is helpful to be defined.